lib/diagnostic.js

```diff
@@ -38,19 +51,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+    name: 'Percy',
+    species: 'Cat',
+    breed: 'Chunky monkey',
+    _vocalization: 'meow',
+
+    vocalize: function (){
+      console.log (this.name + ' says ' + this._vocalization + ' when he wants a butt scratch.');
+    };
```

Comment:
instead of `console.log` you should `return` that string. Also the semi colon at the end of the function is a syntax error, because this function is a property of the pet object. So you could replace that semi colon with nothing or a `,`